.github/workflows/canary.yaml

```diff
@@ -36,37 +36,30 @@ jobs:
           echo $GITHUB_ENV
 
       - name: Install conda environment
-        if: ${{ env.new_off_xml == 'true' }}
+        # if: ${{ env.new_off_xml == 'true' }}
         uses: mamba-org/setup-micromamba@v1
         with:
           environment-file: devtools/conda-envs/canary_env.yaml
           create-args: >-
             python=${{ matrix.python-version }}
 
-      - name: Environment Information
-        if: ${{ env.new_off_xml == 'true' }}
-        shell: bash -l {0}
-        run: |
-          conda info
-          conda list
-
```

Comment:
This is now unnecessary since the installation step actually runs `micromamba info` and `micromamba list` anyway.